lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -47,18 +47,17 @@ class IntegrationTestsManager {
   }
 
   Future<bool> _runPubGet(String workingDirectory) async {
-    final String executable = isCirrus ? environment.pubExecutable : 'flutter';
-    final List<String> arguments = isCirrus
-        ? <String>[
-            'get',
-          ]
-        : <String>[
-            'pub',
-            'get',
-          ];
+    if (isCi) {
+      await _cloneFlutterRepo();
+    }
+    final String executable =
+        isCi ? environment.flutterCommandDir.path : 'flutter';
```

Comment:
When running `flutter` commands, we should always pass the `--local-engine` option. Otherwise, it will use a version of engine that's different from the one that's being tested.

Reply:
done.